pkgs/development/python-modules/gidgethub/default.nix

```
@@ -2,21 +2,21 @@
 , buildPythonPackage
 , fetchPypi
 , pythonOlder
-, setuptools
-, pytest-runner
-, pytest
-, pytest-asyncio
-, twisted
-, treq
-, tornado
-, aiohttp
+, flit
 , uritemplate
 , pyjwt
+, pytestCheckHook
+, aiohttp
+, httpx
+, importlib-resources
+, pytest-asyncio
+, pytest-tornasync
 }:
 
 buildPythonPackage rec {
   pname = "gidgethub";
   version = "5.1.0";
+  format = "pyproject";
```

Doesn't this work instead?
```suggestion
  format = "flit";
```

---

Of course, if this work you should remove `flit` from `propagatedBuildInputs`.

---

Ah, no idea flit was a format. Let me try it.